### PR TITLE
Add update time to db and poll updates more frequently

### PR DESCRIPTION
**Pull request by johnoliver**
# Checklist

- [ ] You added tests to cover the change
- [ ] `mvn clean install` build and test completes
- [ ] You changed or added to the documentation

### CODE REVIEW

File: adoptopenjdk-api-v3-persistance/src/main/kotlin/net/adoptopenjdk/api/v3/dataSources/APIDataStore.kt [hunk 1]
```diff
@@ -69,17 +72,26 @@ object APIDataStore {
...
+                    .map { version ->
+                        ApiPersistenceFactory.get().readReleaseData(version)
+                    }
+                    .filter { it.releases.nodes.isNotEmpty() }
+                    .toList()
+                updatedAt = ApiPersistenceFactory.get().getUpdatedAt()
+
+                val newData = AdoptRepos(data)
+                showStats(binaryRepos, newData)
+                newData
```
Comment (karianna) [inline adoptopenjdk-api-v3-persistance/src/main/kotlin/net/adoptopenjdk/api/v3/dataSources/APIDataStore.kt:91]: Not sure what that line does...

File: adoptopenjdk-api-v3-persistance/src/main/kotlin/net/adoptopenjdk/api/v3/dataSources/APIDataStore.kt [hunk 2]
```diff
@@ -69,17 +72,26 @@ object APIDataStore {
...
+            } else {
+                binaryRepos
```
Comment (karianna) [inline adoptopenjdk-api-v3-persistance/src/main/kotlin/net/adoptopenjdk/api/v3/dataSources/APIDataStore.kt:93]: likewise

File: adoptopenjdk-api-v3-updater/src/main/kotlin/net/adoptopenjdk/api/v3/V3Updater.kt [hunk 3]
```diff
@@ -13,13 +10,30 @@ import net.adoptopenjdk.api.v3.dataSources.persitence.ApiPersistence
 import net.adoptopenjdk.api.v3.models.Variants
 import net.adoptopenjdk.api.v3.stats.StatsInterface
 import org.slf4j.LoggerFactory
+import java.security.MessageDigest
+import java.util.*
```
Comment (karianna) [inline adoptopenjdk-api-v3-updater/src/main/kotlin/net/adoptopenjdk/api/v3/V3Updater.kt:14]: explict imports

File: adoptopenjdk-api-v3-updater/src/main/kotlin/net/adoptopenjdk/api/v3/V3Updater.kt [hunk 4]
```diff
@@ -13,13 +10,30 @@ import net.adoptopenjdk.api.v3.dataSources.persitence.ApiPersistence
...
     private val statsInterface: StatsInterface
 
+    init {
+        val variantData = this.javaClass.getResource("/JSON/variants.json").readText()
+        variants = UpdaterJsonMapper.mapper.readValue(variantData, Variants::class.java)
+        database = ApiPersistenceFactory.get()
+        repo = try {
+            APIDataStore.loadDataFromDb(true)
+        } catch (e: java.lang.Exception) {
+            AdoptRepos(emptyList())
```
Comment (karianna) [inline adoptopenjdk-api-v3-updater/src/main/kotlin/net/adoptopenjdk/api/v3/V3Updater.kt:32]: log error

File: adoptopenjdk-api-v3-updater/src/main/kotlin/net/adoptopenjdk/api/v3/V3Updater.kt [hunk 5]
```diff
@@ -28,18 +42,34 @@ class V3Updater {
...
+
+                        database.updateAllRepos(repo, checksum)
+                        ReleaseVersionResolver.updateDbVersion(repo)
+                        LOGGER.info("Incremental update done")
+                    }
+                    return@runBlocking updatedRepo
+                } catch (e: Exception) {
+                    LOGGER.error("Failed to perform incremental update", e)
+                }
+                repo
```
Comment (karianna) [inline adoptopenjdk-api-v3-updater/src/main/kotlin/net/adoptopenjdk/api/v3/V3Updater.kt:64]: is this a default return?

Comment (johnoliver) [reply adoptopenjdk-api-v3-updater/src/main/kotlin/net/adoptopenjdk/api/v3/V3Updater.kt:64]: yes it is

File: adoptopenjdk-api-v3-persistance/src/main/kotlin/net/adoptopenjdk/api/v3/dataSources/APIDataStore.kt (hunk 1 above)
Comment (johnoliver) [reply adoptopenjdk-api-v3-persistance/src/main/kotlin/net/adoptopenjdk/api/v3/dataSources/APIDataStore.kt:91]: it is the value returned from the statement

File: adoptopenjdk-api-v3-persistance/src/main/kotlin/net/adoptopenjdk/api/v3/dataSources/APIDataStore.kt (hunk 2 above)
Comment (johnoliver) [reply adoptopenjdk-api-v3-persistance/src/main/kotlin/net/adoptopenjdk/api/v3/dataSources/APIDataStore.kt:93]: same

**Pull request merged**